src/adapters.js

```diff
@@ -98,6 +114,9 @@ export function createProjectScaffold(path) {
 }
 
 export function getFilesInFolder(path) {
+  if (!getAccessTokenFromSessionStorage()) {
```

Comment:
is this needed? the consumer of this function is probably assuming it returns a Promise